test/test_integration_list.py

```diff
@@ -0,0 +1,324 @@
+# -*- coding: utf-8 -*-
+
+"""
+CLI List Integration tests
+
+This test file handles the main test flows for the list command
+"""
+import io
+from contextlib import redirect_stderr
+from unittest.mock import patch
+
+from .util.cli_helpers import integration_test
+from test.util.test_runners.integration_test_case import IntegrationTestCaseBase
+
+
+# Not coverage tested since integration tests doesn't run in
+# the same build step
+class CliIntegrationListTest(IntegrationTestCaseBase):  # pragma: no cover
+    DEFINED_VARIABLE_ID = 'one/password'
+    capture_stream = io.StringIO()
+
+    def __init__(self, testname, client_params=None, environment_params=None):
+        super(CliIntegrationListTest, self).__init__(testname, client_params, environment_params)
+
+    # *************** HELPERS ***************
+
+    def setup_cli_params(self, env_vars, *params):
+        self.cli_auth_params = ['--debug']
+        self.cli_auth_params += params
+
+        return self.cli_auth_params
+
+    @patch('builtins.input', return_value='yes')
+    def init_to_cli(self, mock_input):
+        self.invoke_cli(self.cli_auth_params,
+                        ['init', '-u', self.client_params.hostname, '-a', self.client_params.account], exit_code=0)
+
+    def login_to_cli(self):
+        self.invoke_cli(self.cli_auth_params,
+                ['login', '-n', self.client_params.login, '-p', self.client_params.env_api_key], exit_code=0)
+
+    def setUp(self):
+        self.params = self.setup_cli_params({})
+        # Need to configure the CLI and login to perform further commands
+        self.init_to_cli()
+        self.login_to_cli()
```

Comment:
I see we use this two functions in all integration tests, maybe we can move them to the base class for less code repetition

Reply:
would like your help with this because the base path provider doesnt have invoke cli